onnx/defs/schema.cc

```diff
@@ -622,6 +622,38 @@ void OpSchema::ParseAndSetTypes(
   }
 }
 
+OpSchema& OpSchema::FunctionBody(const std::vector<NodeProto>& func_nodes) {
```

Comment:
Can we avoid copies if we use a non-reference parameter "std::vector<NodeProto> func_nodes" and do a std::move? Just wondering what is the best way to avoid some of the needless copies of nodes.